### PR TITLE
Add SH for belgium

### PR DESCRIPTION
**Pull request by pietervdvn**
Note: I'm a bit unsure about this PR, this is my first time trying the format. `npm run test` did finish without problems, but I didn't do interactive testing specifically for Belgium as I didn't get that working.

Let me know if there are still things to improve.

### CODE REVIEW

File: holidays/be.yaml [hunk 1]
```diff
@@ -12,3 +11,33 @@ PH:
   - {'name': 'Allerheiligen - Toussaint', 'fixed_date': [11, 1]}
   - {'name': 'Wapenstilstand - Armistice', 'fixed_date': [11, 11]}
   - {'name': 'Kerstmis - Noël', 'fixed_date': [12, 25]}
+
+# Source: https://nl.wikipedia.org/wiki/Schoolvakantie
```
Comment (ypid) [inline holidays/be.yaml:15]: The `_nominatim_url` is needed. You already had it, but with German coordinates as it seems. See https://github.com/opening-hours/opening_hours.js/tree/master/holidays for details.

File: holidays/be.yaml [hunk 2]
```diff
@@ -1,5 +1,4 @@
 ---
-
 # https://nl.wikipedia.org/wiki/Feestdagen_in_Belgi%C3%AB
```
Comment (ypid) [inline holidays/be.yaml:2]: You don’t like the empty line to separate the YAML header? When I run the Unix command `head -n 2 *.yaml` over the directory, it shows me that all files except `ch.yaml` have an empty line. If you want to change it, you can change it for all files for consistency. I don’t have a strong preference.

**Review by ypid — CHANGES_REQUESTED** [review]
Thanks for your PR. Seem my inline feedback.

File: holidays/be.yaml (hunk 2 above)
Comment (pietervdvn) [reply holidays/be.yaml:5]: That was a little accident. I readde the newline and added the newline in ch.yaml as well

File: holidays/be.yaml (hunk 1 above)
Comment (pietervdvn) [reply holidays/be.yaml:18]: I've added in the style of the ch.yaml which just requests 'Belgium'

**Comment by pietervdvn** [comment]
Before you merge, I still have a question. A public holiday implies that the schools are off too. Is that something that has to be stated too, or is this already implicit?

**Comment by ypid** [comment]
Works now. Please check if the generated school holidays are correct for the next 2 years. The cut exactly at the calendar year boundary is expected in this export:

```Shell
./PH_SH_exporter.js --from 2021 --to 2022 /tmp/out -c be -s -v
2021-01-01--2021-01-03 Kerstvakantie - Vacances de Noël
2021-02-15--2021-02-21 Krokusvakantie - Vacances de carnaval
2021-04-05--2021-04-18 Paasvakantie - Vacances de pâques
2021-07-01--2021-08-31 Zomervakantie - Vacances d'été
2021-11-01--2021-11-07 Herfstvakantie - Vacances automne
2021-12-27--2022-01-09 Kerstvakantie - Vacances de Noël
2022-02-28--2022-03-06 Krokusvakantie - Vacances de carnaval
2022-04-04--2022-04-18 Paasvakantie - Vacances de pâques
2022-07-01--2022-08-31 Zomervakantie - Vacances d'été
2022-10-31--2022-11-06 Herfstvakantie - Vacances automne
2022-12-26--2022-12-31 Kerstvakantie - Vacances de Noël
```

> A public holiday implies that the schools are off too. Is that something that has to be stated too, or is this already implicit?

My understanding of opening_hours in OSM and my implementation of it is that SH and PH are separate. School opening hours would be expressed like this:

Mo-Fr 07:10-18:00; PH,SH off

You can test this here: https://openingh.openstreetmap.de/evaluation_tool/

**Comment by pietervdvn** [comment]
Looks good then

**Comment by ypid** [comment]
Merged and deployed.

**Comment by pietervdvn** [comment]
Sweet! Thanks!

**Comment by pietervdvn** [comment]
Small question: how do users of the npm-library get the latest updates?

**Pull request closed without merging**